starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,9 +3,13 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // TODO: DONE Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
+  articleController.initArticleView = function() {
```

Comment:
You missed one part of the TODO:
Setup a function that kicks off the fetching and rendering of articles, using the same code that used to be in index.html. We need to call a method that fetches our data... :)